yarn.lock

```diff
@@ -131,6 +131,9 @@
     protobufjs "^6.8.9"
     tronweb "^2.7.2"
 
+"@bitgo/statics@../statics":
```

Comment:
Please make sure to regenerate `yarn.lock` after fixing the issue I commented on in the package.json.